Spigot-Server-Patches/0674-Fix-NotePlayEvent.patch

```diff
@@ -0,0 +1,44 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Kieran Wallbanks <kieran.wallbanks@gmail.com>
+Date: Tue, 9 Feb 2021 11:08:46 +0000
+Subject: [PATCH] Fix NotePlayEvent
+
+
+diff --git a/src/main/java/net/minecraft/server/BlockNote.java b/src/main/java/net/minecraft/server/BlockNote.java
+index df69d00d3a38417e53f433cd1eb1f6cf3ec9b55b..415259a95cc7630e15d3a30a08d983b88c644de6 100644
+--- a/src/main/java/net/minecraft/server/BlockNote.java
++++ b/src/main/java/net/minecraft/server/BlockNote.java
+@@ -39,10 +39,13 @@ public class BlockNote extends Block {
+     private void play(World world, BlockPosition blockposition, IBlockData data) { // CraftBukkit
+         if (world.getType(blockposition.up()).isAir()) {
+             // CraftBukkit start
+-            org.bukkit.event.block.NotePlayEvent event = org.bukkit.craftbukkit.event.CraftEventFactory.callNotePlayEvent(world, blockposition, data.get(BlockNote.INSTRUMENT), data.get(BlockNote.NOTE));
+-            if (!event.isCancelled()) {
++            // Paper start - move NotePlayEvent call to fix instrument/note changes
```

Comment:
This comment and the Paper - end comment should be enough. No need for dead code flying around.